### PR TITLE
test: bytecode Emerge

### LINKED ISSUE #221: [compiler] Implement TestBytecode_Emerge()
**Subject**
compiler

**Detail**
Modify `TestBytecode_Emit(t *testing.T)` to `TestBytecode_Emerge(t *testing.T)`!
Then, implement `TestBytecode_Emerge(t *testing.T)` test case in `translate/bytecode_test.go`

### PR DESCRIPTION
**Pull request by zeroFruit**
resolved: #221 

details:

1. test not overflowing digit for operands
2. test overflowing digit for operands

- [x] Test case

### CODE REVIEW

**Review by boohyunsik — COMMENTED** [review]
Testing function names are different in each package.
How about talking about using same function name?

**Comment by zeroFruit** [comment]
~~I would fix that 👍~~ 

done!

File: translate/bytecode_test.go [hunk 1]
```diff
@@ -16,9 +16,127 @@
...
+				{0xff},
+				{0xff},
+			},
+			result: translate.Bytecode{
+				RawByte: []byte{byte(opcode.Mload), 0xff, 0xff},
+				AsmCode: []string{"Mload", "ff", "ff"},
+			},
+		},
+		// How to deal with byte overflow
+		{
```
Comment (hihiboss) [inline translate/bytecode_test.go:67]: Does `Loadfunc` need only 1 operand?

Comment (zeroFruit) [reply translate/bytecode_test.go:67]: Yes I intentionally put 2 operands to show that num of operands is not check here

If we add checking logic to this func, these tcs should be blocked

**Review by boohyunsik — APPROVED** [review]
Good

**Pull request merged**